exec/vector/src/main/java/org/apache/drill/exec/vector/complex/MapVector.java

```diff
@@ -134,10 +134,6 @@ public int getBufferSizeFor(final int valueCount) {
 
   @Override
   public DrillBuf[] getBuffers(boolean clear) {
```

Comment:
perhaps replace with:
```
assert getBufferSize() == super.getBufferSize();
```

We still get the benefit of the check in tests; but the assert is ignored in production runs. This is, in fact, the advantage of the Java `assert` statement over the Guava `Preconditions` methods.